api/promoter.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/SkynetLabs/skynet-accounts/database"
+	"github.com/julienschmidt/httprouter"
+	"gitlab.com/NebulousLabs/errors"
+)
+
+type (
+	// PromoterSetTierPOST describes the body of a POST request that sets the
+	// user's tier.
+	PromoterSetTierPOST struct {
+		Tier int `json:"tier"`
+	}
+)
+
+// _promoterSetTierPOST sets the given user's tier.
+func (api *API) _promoterSetTierPOST(_ *database.User, w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
```

Comment:
Is there a reason for this new convention? The `_`? Gives me very pythony vibes. Like "don't ever use this".

Reply:
I wanted to do it for the endpoints themselves, so it's painfully obvious that those are internal and should never be exposed. Then I backtracked on that and the convention remained for the handlers. But that doesn't make much sense now. I'll remove it.